<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAutolevel</class>
 <widget class="QDialog" name="DialogAutolevel">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autolevel Generation</string>
  </property>
  <widget class="QGroupBox" name="gbAutoleveller">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>113</y>
     <width>490</width>
     <height>287</height>
    </rect>
   </property>
   <property name="title">
    <string>Autoleveller Options</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid Size :</string>
    </property>
    <property name="buddy">
     <cstring>txtGridSize</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="txtGridSize">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>90</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-999.999900000000025</double>
    </property>
    <property name="maximum">
     <double>999.999900000000025</double>
    </property>
    <property name="singleStep">
     <double>0.000100000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>103</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum Depth:</string>
    </property>
    <property name="buddy">
     <cstring>txtMaxDepth</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="txtMaxDepth">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>120</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-999.999900000000025</double>
    </property>
    <property name="maximum">
     <double>999.999900000000025</double>
    </property>
    <property name="singleStep">
     <double>0.000100000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>29</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Units:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbInches">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>66</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Inches</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbMM">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>45</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Milimiters</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>476</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Traverse Height:</string>
    </property>
    <property name="buddy">
     <cstring>txtTraverseHeight</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="txtTraverseHeight">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>150</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-999.999900000000025</double>
    </property>
    <property name="maximum">
     <double>999.999900000000025</double>
    </property>
    <property name="singleStep">
     <double>0.000100000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="txtEngDepth">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>210</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-999.999900000000025</double>
    </property>
    <property name="maximum">
     <double>999.999900000000025</double>
    </property>
    <property name="singleStep">
     <double>0.000100000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>108</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Engraving Depth:</string>
    </property>
    <property name="buddy">
     <cstring>txtEngDepth</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="txtTraverseSpeed">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>180</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-999.999900000000025</double>
    </property>
    <property name="maximum">
     <double>999.999900000000025</double>
    </property>
    <property name="singleStep">
     <double>0.000100000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Traverse Speed:</string>
    </property>
    <property name="buddy">
     <cstring>txtTraverseSpeed</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>240</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Probe Z Speed :</string>
    </property>
    <property name="buddy">
     <cstring>txtProbeZSpeed</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="txtProbeZSpeed">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>240</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-999.999900000000025</double>
    </property>
    <property name="maximum">
     <double>999.999900000000025</double>
    </property>
    <property name="singleStep">
     <double>0.000100000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lblGridSizeUnit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>95</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>inches</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMaxDepthUnit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>125</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>inches</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTravHeightUnit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>155</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>inches</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTravSpeedUnit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>185</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>inches/minute</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEngravingUnit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>215</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>inches</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProbeZSpeedUnit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>245</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>inches/minute</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input file :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>37</y>
     <width>73</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output file :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtInputFile">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>5</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBrowseInputFile">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>5</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtOutputFile">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>35</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBrowseOutputFile">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>35</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>514</x>
     <y>128</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Autolevel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>514</x>
     <y>158</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProgress">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>418</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Progress ...</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb1">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>445</y>
     <width>486</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblBoardSize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>536</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Board Size: </string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtInputFile</tabstop>
  <tabstop>pbBrowseInputFile</tabstop>
  <tabstop>txtOutputFile</tabstop>
  <tabstop>pbBrowseOutputFile</tabstop>
  <tabstop>rbInches</tabstop>
  <tabstop>rbMM</tabstop>
  <tabstop>txtGridSize</tabstop>
  <tabstop>txtMaxDepth</tabstop>
  <tabstop>txtTraverseHeight</tabstop>
  <tabstop>txtTraverseSpeed</tabstop>
  <tabstop>txtEngDepth</tabstop>
  <tabstop>txtProbeZSpeed</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogAutolevel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogAutolevel</receiver>
   <slot>pbAutolevelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbBrowseOutputFile</sender>
   <signal>clicked()</signal>
   <receiver>DialogAutolevel</receiver>
   <slot>pbBrowseOutputFileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>pbAutolevelClicked()</slot>
  <slot>pbBrowseOutputFileClicked()</slot>
 </slots>
</ui>
